cypress/helpers/createAO.js

```diff
@@ -0,0 +1,54 @@
+import { DIMENSION_ID_EVENT_DATE } from '../../src/modules/dimensionConstants.js'
+import {
+    TEST_REL_PE_LAST_YEAR,
+    TEST_REL_PE_LAST_5_YEARS,
+    WHO_RMNCH_TRACKER_PROGRAM,
+} from '../data/index.js'
+import { addConditions } from './conditions.js'
```

Comment:
since this is only used in this file there's no need to keep it in a separate helper file